common/gratia/common/sandbox_mgmt.py

```diff
@@ -569,11 +569,14 @@ def OpenNewRecordFile(dirIndex):
             if not os.path.exists(working_dir):
                 try:
                     Mkdir(working_dir)
-                except:
+                except Exception as e:
+                    DebugPrint(0, 'Warning: Exception caught while creating directory: ' + working_dir + ':' + e)
```

Comment:
```suggestion
                except Exception as exc:
                    DebugPrint(0, 'Warning: Exception caught while creating directory: ' + working_dir + ':' + exc)
```

I said I was going to be annoying about PEP8